src/vma/proto/mem_buf_desc.h

```diff
@@ -123,7 +124,8 @@ class mem_buf_desc_t {
 			size_t		sz_payload;
 
 			struct timespec sw_timestamp;
```

Comment:
If rx sw timestamp is enabled by the user, it will be calculated in _sockinfo_udp::rx_input_cb_  :
```
if (m_b_rcvtstamp || (m_n_tsing_flags & (SOF_TIMESTAMPING_RX_SOFTWARE | SOF_TIMESTAMPING_SOFTWARE))) {
	clock_gettime(CLOCK_REALTIME, &(p_desc->path.rx.sw_timestamp));
}
```

Then, if the user will request the SW timestamp for the packet, we will copy the content in _sockinfo_udp::handle_recv_timestamping_
```
if (m_n_tsing_flags & SOF_TIMESTAMPING_SOFTWARE) {
	tsing.systime = packet->path.rx.sw_timestamp;
}
```
i dont think we can remove it from _mem_buf_desc_t_.